components/playlist/browser/playlist_service.cc

```diff
@@ -319,8 +320,15 @@ void PlaylistService::NotifyPlaylistChanged(
     obs.OnPlaylistStatusChanged(params);
 
   // TODO(sko) Send proper events based on |params|
+  // for (auto& service_observer : service_observers_)
+  //   service_observer->OnEvent(mojom::PlaylistEvent::kUpdated);
+}
+
+void PlaylistService::NotifyPlaylistChanged(mojom::PlaylistEvent playlist_event,
```

Comment:
Having two functions 
```
void PlaylistService::NotifyPlaylistChanged(
    const PlaylistChangeParams& params)
    
void PlaylistService::NotifyPlaylistChanged(mojom::PlaylistEvent playlist_event, const std::string& playlist_id);
```
with the same name really confuses.

Especially when they are invoked close each to other
```

  NotifyPlaylistChanged({mojom::PlaylistEvent::kItemUpdated, item->id});
  NotifyPlaylistChanged(mojom::PlaylistEvent::kItemUpdated, item->id);
```

Similar happens in all methods, I count 11 places,  except `PlaylistService::MoveItem`. 

Is it possible to re-arrange this?